src/test/java/K_pages/AllSongsPage.java

```diff
@@ -0,0 +1,40 @@
+package K_pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+public class AllSongsPage extends BasePage{
+ @FindBy(css=".playlist:nth-child(3)")
+WebElement playlistLocator;
+protected @FindBy(css="input[name='name']")
```

Comment:
How can I beautify code in IntelliJ:

The formatting shortcuts in Intellij IDEA are :
For Windows : Ctrl + Alt + L.
For Ubuntu : Ctrl + Alt + Windows + L.
For Mac : ⌥ (Option) + ⌘ (Command) + L.

![image](https://user-images.githubusercontent.com/104761482/209774134-9e63bfed-d768-4455-a860-028305c85e17.png)